crates/typst/src/lib.rs

```diff
@@ -12,8 +12,8 @@
 //!   the source file. The elements of the content tree are well structured and
 //!   order-independent and thus much better suited for further processing than
 //!   the raw markup.
-//! - **Layouting:**
-//!   Next, the content is [layouted] into a [document] containing one [frame]
+//! - **Layout:**
+//!   Next, the content is [laid-out] into a [document] containing one [frame]
```

Comment:
Reading this again, I'm realizing that this should in fact correspond to the item below and that I've messed it up.